test-data/user/types.ts

```diff
@@ -1,4 +1,5 @@
 import type { TBuilder } from '@commercetools-test-data/core';
+import { TVerificationStatus } from '../../packages/application-shell/src/types/generated/mc';
```

Comment:
It does seem like a good idea to import directly from autogenerated types, but I'm unsure about importing these types from distant directories, is there a better way to do this?